src/core/config_mod.F90

```diff
@@ -758,7 +758,8 @@ LOGICAL FUNCTION is_int(this, key)
         CHARACTER(len=*), INTENT(in) :: key
 
         ! https://github.com/nlohmann/json/blob/develop/include/nlohmann/detail/value_t.hpp
-        is_int = this%is_type(key, 6)
+        is_int = this%is_type(key, 5) ! = signed int
+        ! is_int = this%is_type(key, 6) ! = unsigned int
```

Comment:
I do not like lingering commented out code around. In particular when it is incorrect. The commented would be best removed.